tweak code samples to avoid issues with CRAN

nikosbosse opened the pull request:
Sets number of threads to 1 for examples

nikosbosse commented:
Hm curious... this does not fail on my local machine.. Will investigate

nikosbosse commented:
still confused.. @Bisaloo do you maybe have an idea? It seems like the snapshot review fails, but there is nothing that I can update on my local machine

seabbs commented:
I can look at this later but we have a solution for the original problem in `EpiNow2` I think

seabbs commented:
As a test does the `vdiffr` issue happen for any PR or just this one?

nikosbosse commented:
apparently this was due to an outdated version of an upstream package on my machine. Updated, recreated the plots, updated the snapshots and now works like a charm
